annotator-models/README.md

```diff
@@ -0,0 +1,57 @@
+# Modeling Anotators
+
+This is an implementation of the Dawid-Skene algorithm to model annotator error rate and get predictions of true comment labels.
+
+## To Run Locally
+
+1.  Setup a (virtualenv)[https://virtualenvwrapper.readthedocs.io/en/latest/] for
+    the project (recommended, but technically optional).
+
+    Python 2:
+
+    ```
+    python -m virtualenv env
+    ```
+
+    Python 3:
+
+    ```
+    python3 -m venv env
+    ```
+
+    From either to enter your virtual env:
+
+    ```shell
+    source env/bin/activate
+    ```
+
+2.  Install library dependencies:
+
+    ```shell
+    pip install -r requirements.txt
+    ```
+
+3.  Create a CSV file of data to train on. The data must be a CSV that has fields:
+    * `_unit_id` corresponding to the item id
```

Comment:
Confirm that this is the correct id field from dataset.

Reply:
confirmed :)